### PR TITLE
Kanban Capstone: Add comment counter

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
# Kanban Capstone: Add comment counter

In this project, I
- add a comment counter
- implement add comment
- implement get the comments
- display comment count
- implement jest testing

### CODE REVIEW

File: src/Meals.js [hunk 1]
```diff
@@ -73,6 +95,28 @@ const mealPopUp = async (meal) => {
...
+    const comment = document.querySelector('#comment').value;
+
+    const mealId = meal.idMeal;
+
+    const newComment = {
+      item_id: mealId,
+      username,
+      comment,
+    };
+    addComment(newComment);
```
Comment (lu-jim) [inline src/Meals.js]: Consider validating the input so that there are no empty users/comments

File: src/index.html [hunk 2]
```diff
@@ -28,14 +29,8 @@
...
     </div>
   </nav>
-<<<<<<< HEAD
-  <section class="container d-flex my-4 py-3 flex-wrap justify-content-evenly g-4  border border-dark border-3">
-    <div class="row g-3 justify-content-evenly" id="board">
-    </div>
-=======
   <section class="container d-flex my-4 py-3 flex-wrap justify-content-evenly g-4">
     <div class="row g-3 justify-content-between" id="board"></div>
->>>>>>> dev
```
Comment (lu-jim) [inline src/index.html]: Let's address this conflict and keep either `justify-content-evenly `or use `justify-content-start`. With `between`, it seems to me like there is an item missing in the center.
image

File: src/css/styles.css [hunk 3]
```diff
@@ -158,19 +158,25 @@ body {
   opacity: 0.8;
 }
 
-.list-comment {
```
Comment (lu-jim) [inline src/css/styles.css:161]: Consider removing the font import and usage in line 1 and from lines 14-19, as we are not making use of this code.

**Review by lu-jim — CHANGES_REQUESTED** [review]
Good effort @Emmy-github-webdev 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
Let me know if there is anything you would like to improve in the project so we can do it together.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/Meals.js (hunk 1 above)
Comment (Emmy-github-webdev) [reply src/Meals.js]: Required not taking effect

File: src/index.html (hunk 2 above)
Comment (Emmy-github-webdev) [reply src/index.html]: Fixed

File: src/css/styles.css (hunk 3 above)
Comment (Emmy-github-webdev) [reply src/css/styles.css:161]: . list-comment is in use in the pop up modal

image

Comment (lu-jim) [reply src/css/styles.css:161]: I placed the comment here as it didn't let me comment above. That is why I specified that my request was for line 1 and lines 14-19
image

Comment (Emmy-github-webdev) [reply src/css/styles.css:161]: Highlighted removed

File: src/Meals.js (hunk 1 above)
Comment (lu-jim) [reply src/Meals.js]: I just pushed a small implementation for this. Please review and let me know if you need any changes.

Comment (Emmy-github-webdev) [reply src/Meals.js]: Go ahead to merge

**Comment by lu-jim** [comment]
@Emmy-github-webdev 

Your feature is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Review by lu-jim — APPROVED** [review]

**Comment by Emmy-github-webdev** [comment]
Thank you @lu-jim

**Pull request merged**